src/MainUtils.hpp

```diff
@@ -37,20 +37,30 @@
 #include "mlir/Target/LLVMIR.h"
 #include "mlir/Transforms/Passes.h"
 
+namespace onnx_mlir {
+extern std::string kExecPath;
```

Comment:
Is this actually needed? Seems like only reference to this variable are inside MainUtil.cpp, and in MainUtil.cpp, ExternalUtil.hpp is actually included, which contains the definition.

Reply:
I was hoping that we don't have to do this but unfortunately not everybody includes `ExternalUtil.hpp`. For example, `TestConv.cpp` includes `MainUtil.hpp` but not `ExternalUtil.hpp`. If there is some other better way to "fix" this, feel free to suggest.

Reply:
Hmm...I think we can indeed remove it from `MainUtils.hpp`. At some point I was setting `kExecPath` directly in `main()` so `TestConv.cpp` won't compile without the `extern std::string kExecPath`. Then I made a `setExecPath()` call in `MainUtils.cpp` to do that and forgot that now we can indeed remove the `extern`.

Thanks for catching it!

Reply:
Sounds good!